www/configurator/src/index.js

```diff
@@ -33,36 +33,63 @@ if (process.env.NODE_ENV === 'production') {
   cssPath = 'test-dist/uncompiled-css/templates/';
 }
 
+// Function to get params on the page
+function getUrlParams() {
+  const params = {
+    search: {},
+    hash: {}
+  };
+  const templateParam = queryString.parse(location.search.substring(1)).template;
+  if (templateParam) {
+    params.search.template = templateParam;
+  }
+  params.hash = Object.assign({}, params.hash, queryString.parse(location.hash.substring(1)));
+  return params;
+}
+
+// Define our iframe manager, an our css transplier
+let iframeManager = false;
+let cssTranspiler = false;
+// Get our url params on page load
+let params = getUrlParams();
+console.log('Initial Params', params);
+
 // Define our hash change handler
 function handleHashChange_() {
   console.log('Hash Changed, re-building template...');
-  params = queryString.parse(location.hash.substring(1));
+  params = getUrlParams();
+  const updatedStyles = cssTranspiler.getCssWithVars(params.hash);
+  iframeManager.setStyle(updatedStyles);
+  console.log('New params:', params);
 }
-// Handle the beginning hash change, and our event listener
-handleHashChange_();
-window.addEventListener('hashchange', handleHashChange_);
-
-// require('postcss-custom-properties')({preserve: true}),
 
 // Grab our CSS and CSS Json
-const cssRequests = [];
-const templateCssPath = `${cssPath}${params.template}/page`;
-cssRequests.push(
+const configuratorInit = [];
+const templateCssPath = `${cssPath}${params.search.template}/page`;
+configuratorInit.push(
   fetch(`${templateCssPath}.json`).then(response => {
     return response.json();
   })
 );
-cssRequests.push(
+configuratorInit.push(
   fetch(`${templateCssPath}.css`).then(response => {
     return response.text();
   })
 );
 
-Promise.all(cssRequests).then(responses => {
+// Create the configurator
+const templateSrc = `${templatesPath}${params.search.template}/${params.search.template}.amp.html#amp=1`;
```

Comment:
It's being set underneath the imports. `templatesPath` is conditionally set based on the current environment variable that webpack exposes when it builds